Source/ThirdParty/libwebrtc/Configurations/DebugRelease.xcconfig

```diff
@@ -40,6 +40,8 @@ VALIDATE_DEPENDENCIES_INTERNAL_ = $(VALIDATE_DEPENDENCIES_NATIVE_TARGET_$(WK_NOT
 VALIDATE_DEPENDENCIES_INTERNAL_YES = $(WK_VALIDATE_DEPENDENCIES);
 VALIDATE_DEPENDENCIES_NATIVE_TARGET_YES = $(WK_VALIDATE_DEPENDENCIES);
 
+WK_BUILD_FOR_TESTING = YES;
```

Comment:
I don't think that I agree with the idea of DebugRelease being equal to "build for testing". That's a different concept in Xcode, so we'd be overloading the term here.

Reply:
This matches what we already do in WebKitLegacy.

Should I just use the Configuration to enable this mode with a different Xcode variable name that doesn’t include “testing”?

Also, do we actually run tests on any builds other than Debug And Release configs?

Reply:
I changed this to `WK_EXPORT_SYMBOLS_FOR_TESTING` since there was no specific feedback given about a different name.  I will assume this is okay unless you say otherwise.